test/unit/alignment/pairwise/align_pairwise_test.cpp

```diff
@@ -177,11 +175,10 @@ TYPED_TEST(align_pairwise_test, single_pair_double_score)
             }
         }
 
-        {  // the alignment
+        {  // with everything (default if no output is specified)
             seqan3::configuration cfg = seqan3::align_cfg::method_global{} |
                                         seqan3::align_cfg::edit_scheme |
-                                        seqan3::align_cfg::result{seqan3::with_alignment,
-                                                                  seqan3::using_score_type<double>};
+                                        seqan3::align_cfg::result{seqan3::with_score, seqan3::using_score_type<double>};
```

Comment:
This also seems to be a TODO. Can you maybe add all these lines to the respective issue so we don't forget about them?

Reply:
This won't be necessary, because when the PR with the score_type is through the result config will be deleted and all the tests wouldn't compile.

Reply:
:smile: well thats true. You have many TODO comments in this PR though, so I thought you deliberately did this..

Reply:
It was more or less the places that I had to touch but couldn't resolve completely so I marked them. But not all places seem to be affected by my change so I didn't see this.